### PR TITLE
Fix CR/LF using dos2unix utility

### PR DESCRIPTION
**Pull request by dims**
Follow up from https://github.com/kubernetes/kubernetes/pull/127713/#issuecomment-2383191880

image

### CODE REVIEW

**Review by akhilerm — COMMENTED** [review]
Screenshot from 2024-09-30 19-13-26

Github complains with this error in CODEOWNERS file. Is this expected?

**Comment by dims** [comment]
@akhilerm it was there before, i am not touching it :) the repo owners an update it in another PR hopefully :) 

(and they would ensure they don't flip the CRLF back on!! when they do so)

**Comment by liggitt** [comment]
would adding a root .gitattributes file pinning EOL to LF help as well?

e.g. https://github.com/kubernetes/kubernetes/pull/114388

**Comment by dims** [comment]
> would adding a root .gitattributes file pinning EOL to LF help as well?

good point @liggitt doing that now!

File: CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  - debjitmondal@microsoft.com
-  - daschott@microsoft.com
+# Owner docs for hnslib
+
+approvers:
+  - sbangari@microsoft.com
+  - souravdas@microsoft.com
+  - ppereira@microsoft.com
+  - debjitmondal@microsoft.com
+  - daschott@microsoft.com
```
Comment (princepereira) [inline CODEOWNERS]: Please remove these error lines as it is complaining. Will add it later

**Review by princepereira — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (dims) [reply CODEOWNERS]: done

**Comment by kiashok** [comment]
lgtm

**Review by kiashok — APPROVED** [review]

**Pull request merged**